content/intro-to-storybook/svelte/es/using-addons.md

```diff
@@ -0,0 +1,188 @@
+</div>
+
+Podríamos escribir para siempre sobre la configuración y el uso de complementos para todos sus casos de uso particulares. Por ahora, trabajemos para integrar uno de los complementos más populares dentro del ecosistema de Storybook: [knobs](https://github.com/storybooks/storybook/tree/master/addons/knobs).
+
+## Configurando Knobs
+
+Knobs es un recurso increíble para que los diseñadores y desarrolladores experimenten y jueguen con componentes en un entorno controlado sin necesidad de codificar! Básicamente, proporciona campos definidos dinámicamente con los que un usuario manipula los props que se pasan a los componentes de sus historias. Esto es lo que vamos a implementar ...
+
+<video autoPlay muted playsInline loop>
+  <source
+    src="/intro-to-storybook/addon-knobs-demo.mp4"
+    type="video/mp4"
+  />
+</video>
+
+### Instalación
+
+Primero, tendremos que agregarlo como una dependencia de desarrollo
+
+```bash
+npm install -D @storybook/addon-knobs
+```
+
+Registra Knobs en tu archivo `.storybook/main.js`.
+
+```javascript
+// .storybook/main.js
+
+module.exports = {
+  stories: ['../src/components/**/*.stories.js'],
+  addons: ['@storybook/addon-actions', '@storybook/addon-links','@storybook/addon-knobs'],
+};
+
+```
+
+<div class="aside">
+<strong> 📝 ¡El orden de registro de complementos es importante! </strong>
+<br/>
+El orden en que enumere estos complementos determinará el orden en que aparecerán como pestañas en su panel de complementos (para aquellos que aparecen allí).
+</div>
+
+¡Eso es! Es hora de usarlo en una historia.
+
+### Uso
+
+Usemos el tipo de knob de objeto en el componente `Task`.
+
+Primero, importe el decorador `withKnobs` y el tipo de knob `object` a `Task.stories.js`:
+
+```javascript
+// src/components/Task.stories.js
+import { action } from "@storybook/addon-actions";
+import { withKnobs, object } from '@storybook/addon-knobs';
+```
+
+A continuación, dentro de la exportación `default` del archivo`Task.stories`, agregue `withKnobs` como elemento de `decorators`:
+
+```javascript
+// src/components/Task.stories.js
+
+export default {
+  title: 'Task',
+  decorators: [withKnobs],
+  // same as before
+};
+```
+
+Por último, integre el tipo de knob `object` dentro de la historia "predeterminada":
+
+```javascript
+// src/components/Task.stories.js
+export const Default = () => ({
+  Component: Task,
+  props: {
+    task: object("task", { ...taskData })
+  },
+  on: {
+    ...actionsData
+  }
+});
+export const Pinned = () => ({
+  Component: Task,
+  props: {
+    task: {
+      ...taskData,
+      state: "TASK_PINNED"
+    }
+  },
+  on: {
+    ...actionsData
+  }
+});
+export const Archived = () => ({
+  Component: Task,
+  props: {
+    task: {
+      ...taskData,
+      state: "TASK_ARCHIVED"
+    }
+  },
+  on: {
+    ...actionsData
+  }
+});
```

Comment:
@IvanSotelo we can cut down this snippet like we did before so that the reader keeps engaged on the actual change, more specifically in the `Default ` story with the addon. You can check #304.